### DISCUSSION TITLE
How to code the "github repo" - "Showcases" - "Credits" - "tags" button from the example

### DISCUSSION THREAD

**Discussion by vPackets** [McShelby/hugo-theme-relearn#487] [Q&A] [answered]
Hi guys !

I have looked in the documentation and I apologize in advance if it is there but my doc foo seems broken.

as the title says, I am trying to understand how to code these buttons on the left to provide some link to my twitter/github and any other useful links.

Thanks a ton !

**Comment by cagix** [comment, marked as answer]
iirc this is defined in `layouts/partials/menu-footer.html`.

edit: have a look at https://github.com/McShelby/hugo-theme-relearn/blob/main/exampleSite/layouts/partials/menu-footer.html

**Reply by McShelby** [reply]
Your edit is correct. This is the overridden template for the documentation in the exampleSite. The original theme implementation has an empty menu footer.

**Comment by vPackets** [comment]
So I have copied this exact code into my site vPackets-blog/layouts/partials as menu-footer.html but it does not work ... Hmm

Checking what I did wrong

**Comment by McShelby** [comment]
See the 404 in your network tab? This is because you haven't copied the `buttons.js` file over to your site.

**Comment by vPackets** [comment]
Ok I get it and I apologize in advance for the confusion but this is not the menu footer I need help with. 

On the example website and on the left side of the page, you can find the "github repo" "showscases" "hugo"documentation" "Credits" and tags.

That's what I am trying to setup up and was looking at layouts and was trying to understand how to make it happen :D 

PS: I am able to see changes in my menu-footer for what it's worth :)

**Reply by McShelby** [reply]
Thats needs to be defined in your `config.toml`. See the exampleSite's config.toml for an example. See the docs for explaination at https://mcshelby.github.io/hugo-theme-relearn/cont/menushortcuts/index.html

**Reply by vPackets** [reply]
Ah crap ... my Google/doc fu was crap ! I apologize and thank you for your work on this project. it ROCKS
